x-pack/plugins/watcher/public/application/sections/watch_status_page/components/execution_history_panel.tsx

```diff
@@ -135,18 +138,79 @@ export const ExecutionHistoryPanel = () => {
     },
     {
       field: 'watchStatus.state',
-      name: i18n.translate('xpack.watcher.sections.watchHistory.watchTable.stateHeader', {
-        defaultMessage: 'State',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.watcher.sections.watchHistory.watchTable.stateHeader.tooltipText',
+            {
+              defaultMessage: 'Active or error state',
```

Comment:
>Active or error

Should this also include "inactive"?

Reply:
I don't believe so, because this part of the UI shows a history of attempted executed actions, and if it's inactive then it won't execute.